pkg/aws/commandbuilder/helper/roles/roles.go

```diff
@@ -107,6 +119,7 @@ type ManualCommandsForUpgradeAccountRolePolicyInput struct {
 	DefaultPolicyVersion string
 	RoleName             string
 	HasPolicy            bool
+	PolicyAttached       bool
```

Comment:
Can policyAttached by renamed to something like
RequiredRolePolicyAttached?
The difference between HasPolicy and PolicyAttached isn't clear for me. 😢